test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js

```diff
@@ -436,6 +456,13 @@ describe( `[${ host }] Calypso Gutenberg Site Editor Tracking: (${ screenSize })
 				if ( editor.screenSize === 'mobile' ) {
 					return this.skip();
 				}
+				// Reset Global Styles before testing.
+				await clickGlobalStylesResetButton( this.driver );
+				await saveGlobalStyles( this.driver );
+				// Wait for debounce and entity retreival before clearing stack.
+				await this.driver.sleep( 500 );
```

Comment:
In general, fixed `sleep`s are not a good practice as the execution speed differs across environments. While this might be working in a stable manner locally, it will not necessarily be the same for the CI, and vice versa. Is there any element on the page we could wait for instead? I.e. a placeholder disappearing, the save button becoming disabled, etc.

Reply:
🤔  Im not sure the best way to go about it, but waiting for the save UI to adapt won't work either.  With the function to build the events debounced, it wont run until 100ms after that UI updates.  Additionally it will have to get and compare entity records before populating the events stack.

Reply:
Gotcha. Let's keep the sleep then. The important part is that we have the comment that explains why it's been added. Thanks! 👍